testapps/automationapp/build.gradle

```diff
@@ -46,6 +46,6 @@ android {
 }
 
 dependencies {
-    compile project (':msal')
-    compile "com.android.support:appcompat-v7:$rootProject.ext.supportLibraryVersion"
+    implementation project (':msal')
```

Comment:
changing from compile to implementation threw several issues when i was building MSAL, why is this required?

Reply:
I am not too hung up on this, if this builds I am good.

Reply:
This change is coming from the compatibility requirements of common-core submodule. The common-core is using buildToolsVersion = "26.0.2" and gradle v3.0.1. That is why I upgrade the versions here. And compile is deprecated in 26 and it need to be replaced with implementation.